omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PersonAttributeTypeController1_8Test.java

```diff
@@ -153,6 +154,27 @@ public void retirePersonAttributeType_shouldRetireAPersonAttributeType() throws
 		Assert.assertTrue("unit test".equals(obj.getRetireReason()));
 	}
 	
+	@Test
+	public void shouldUnRetireAPersonAttributeType() throws Exception {
+		
+		final String nonRetiredAttribute = "a0f5521c-dbbd-4c10-81b2-1b7ab18330df";
+		PersonAttributeType obj = service.getPersonAttributeTypeByUuid(nonRetiredAttribute);
+		obj.setRetired(true);
+		obj.setRetiredBy(Context.getAuthenticatedUser());
+		obj.setDateRetired(new Date());
+		obj.setRetireReason("random reason");
+		service.savePersonAttributeType(obj);
```

Comment:
What happens if you comment out the above line?

Reply:
This has been resolve.